apis/Google.Cloud.Logging.Console/docs/index.md

```diff
@@ -50,3 +30,27 @@ using Google.Cloud.Logging.Console;
 var builder = WebApplication.CreateBuilder(args);
 builder.Logging.AddGoogleCloudConsole(options => options.IncludeScopes = true);
 ```
+
+## Formatter output
+
+The formatter writes JSON entries in the format expected by Cloud
+Logging. In particular, it uses the following JSON property names:
+
+- `message`: the formatted log message
+- `severity`: the log entry severity
+- `category`: the logger category
+- `exception`: the exception stack trace, if any
+- `format_parameters`: the format parameters for the message, if any
+- `scopes`: the scope information for the log entry, if the
+  `IncludeScopes` option is enabled and there is scope information
+  available
+
+No timestamp is included in the JSON, as this is automatically
+picked up by the logging agent.
+
+Most log entry severity mappings are obvious, but the ASP.NET Core
+`LogLevel.Trace` level maps to `DEBUG` in Cloud Logging.
+
+When viewing the log entries in the Logs Explorer, the severity is
```

Comment:
Just a question to confirm, we now don't have any control on where the entries are shown as the Google Cloud log name is set by the agent, right?
This is fine, and not much we can do about it, but something to take into account as we used to get some questions around this before.

Reply:
At the moment, that's right. I suspect if we start including extra information in the JSON, that would be used - and we've got the options for that when we know what we want to add.